var/spack/repos/builtin/packages/py-torchvision/package.py

```diff
@@ -54,7 +54,7 @@ class PyTorchvision(PythonPackage):
     depends_on('py-torch@1.6.0', when='@0.7.0', type=('build', 'link', 'run'))
     depends_on('py-torch@1.5.1', when='@0.6.1', type=('build', 'link', 'run'))
     depends_on('py-torch@1.5.0', when='@0.6.0', type=('build', 'link', 'run'))
-    depends_on('py-torch@1.4.0', when='@0.5.0', type=('build', 'link', 'run'))
+    depends_on('py-torch@1.4.0:1.4.999', when='@0.5.0', type=('build', 'link', 'run'))
```

Comment:
I would make it 1.4.1 instead, no reason to support 1.4.0 if it's broken. Technically these should all be ranges, but the developers don't seem to want to support that and I don't know what range will work.